### PR TITLE
[CARBONDATA-1514] Sort Column  Property is not getting added in case of alter operation

### PR DESCRIPTION
**Pull request by kumarvishal09**
**Problem:**: Sort Column  Property is not getting added in case of alter operation
**Solution**: Need to add sort column if use has added in alter operation

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/287/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/163/

**Comment by ravipesala** [comment]
SDV Build Fail , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/919/

**Comment by ravipesala** [comment]
@kumarvishal09 SDV testcase is failed, please check

File: integration/spark-common/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchemaCommon.scala [hunk 1]
```diff
@@ -209,7 +217,7 @@ class AlterTableColumnSchemaGenerator(
         -1,
         field.precision,
         field.scale,
-        field.schemaOrdinal + existingColsSize)
+        field.schemaOrdinal + existingColsSize, checkSortColumn(field.name.getOrElse(field.column)))
```
Comment (jackylk) [inline integration/spark-common/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchemaCommon.scala]: each parameter should be in one line

File: integration/spark-common/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchemaCommon.scala [hunk 2]
```diff
@@ -226,7 +234,7 @@ class AlterTableColumnSchemaGenerator(
         -1,
         field.precision,
         field.scale,
-        field.schemaOrdinal + existingColsSize)
+        field.schemaOrdinal + existingColsSize, false)
```
Comment (jackylk) [inline integration/spark-common/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchemaCommon.scala]: each parameter should be in one line

File: integration/spark-common/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchemaCommon.scala [hunk 3]
```diff
@@ -190,6 +190,14 @@ class AlterTableColumnSchemaGenerator(
 
   val LOGGER = LogServiceFactory.getLogService(this.getClass.getName)
 
+  def checkSortColumn(columnName: String): Boolean = {
```
Comment (jackylk) [inline integration/spark-common/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchemaCommon.scala]: change function name to `isSortColumn`

File: integration/spark-common/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchemaCommon.scala [hunk 4]
```diff
@@ -190,6 +190,14 @@ class AlterTableColumnSchemaGenerator(
...
+    val sortColumns = alterTableModel.tableProperties.get("sort_columns")
+    if(sortColumns.isDefined) {
+      sortColumns.get.contains(columnName)
+    } else {
+      true
```
Comment (jackylk) [inline integration/spark-common/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchemaCommon.scala]: Shouldn't it be false?

Comment (kumarvishal09) [reply integration/spark-common/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchemaCommon.scala]: by default sort column is enabled for all the dimension column, if user is not providing any column property then it is true

File: integration/spark-common/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchemaCommon.scala (hunk 3 above)
Comment (kumarvishal09) [reply integration/spark-common/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchemaCommon.scala]: ok

File: integration/spark-common/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchemaCommon.scala (hunk 2 above)
Comment (kumarvishal09) [reply integration/spark-common/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchemaCommon.scala]: ok

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/366/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/242/

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/994/

**Comment by ravipesala** [comment]
LGTM

**Pull request closed without merging**